API Set default value of framework RememberLoginHash logout_across_devices to false (#66)
silverstripe/silverstripe-session-manager: README.md

## Configuration

### Logout across devices

> This module will update the `SilverStripe\Security\RememberLoginHash.logout_across_devices` config setting, setting it to `false` by default.

Reviewer: ```suggestion
This module updates the `SilverStripe\Security\RememberLoginHash.logout_across_devices` configuration flag to default to `false`.
```

Author: Have updated text

This is done because session manager is designed to provide a good way to revoke sessions for devices that you do not want to be logged in.  Without session manager, the only way to revoke a different device was the slightly odd method of logging out of your current device, which in turn deleted the database record created when a user checked the "Remember me" checkbox when logging in to the admin section. This is considered to be an odd UX experience if you are legitimately logged in on multiple devices.

> To restore the old behaviour with session manager installed, add the following YML config to your project:

Reviewer: I don't think we need to go into this level of detail here.

There's a already a [bit about this config flag in the regular doc](https://docs.silverstripe.org/en/4/developer_guides/security/member/#saved-user-logins). If this does not provide enough info, then it probably make more sense to update it there.

I would be tempted to keep it short and sweet. Maybe something like:
> Session-manager provides an explicit way to terminate individual sessions and their attached "remember me" tokens. So this module sets `SilverStripe\Security\RememberLoginHash.logout_across_devices` to `false`.
>
> To restore the old behaviour with session manager installed, add the following YML config to your project:
> ...
> Read [Saved User Logins](https://docs.silverstripe.org/en/4/developer_guides/security/member/#saved-user-logins) to learn how to configure the "remember me" feature for your users.

Author: Updated to your text
